.github/workflows/api-server-tests.yml

```
@@ -106,14 +106,13 @@ jobs:
           image_tag: ${{ needs.build.outputs.IMAGE_TAG }}
           image_registry: ${{ needs.build.outputs.IMAGE_REGISTRY }}
 
-      - name: Configure Cluster CA Cert for TLS-Enabled Tests
+      - name: Configure Cluster CA e2e for TLS-Enabled Tests
         shell: bash
         if: ${{ matrix.pod_to_pod_tls_enabled == 'true'}}
         run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:2:55: Double quote to prevent globbing and word splitting

(shellcheck)